README.md

```diff
@@ -28,6 +28,8 @@ The repo was inspired by the similar one from [Summer 2022 Internships by Pitt C
 | Amazon| EMEA - [Dublin](https://www.amazon.jobs/en/jobs/1694116/software-development-engineer-internship-2022-dublin), [Bucharest](https://www.amazon.jobs/en/jobs/1674285/software-dev-engineer-intern-bucharest-summer-2022), [Madrid](https://www.amazon.jobs/en/jobs/1709009/software-development-engineer-internship-2022-madrid-spain), [Barcelona](https://www.amazon.jobs/en/jobs/1709010/software-development-engineer-internship-2022-barcelona-spain), [Berlin](https://www.amazon.jobs/en/jobs/1729097/software-development-engineer-intern-2022-berlin), [Luxembourg](https://www.amazon.jobs/en/jobs/1729117/software-development-engineer-internship-2022-luxembourg) | SWE Intern, SDE Intern |
 | [Facebook](https://www.facebook.com/careers/v2/jobs/389614179266190/)| EMEA - London | Software Engineer, Intern/Co-op |
 | <del>Google</del> | APAC - Singapore | STEP Internship. **Closed** |
+| Google | EMEA - [Zurich](https://careers.google.com/jobs/results/140736005071086278-student-training-in-engineering-program-step-2022/?degree=BACHELORS&distance=50&employment_type=INTERN&q=STEP), [Paris and Warsaw](https://careers.google.com/jobs/results/128555838596883142-student-training-in-engineering-program-step-2022/?degree=BACHELORS&distance=50&employment_type=INTERN&q=STEP), [Dublin](https://careers.google.com/jobs/results/81989322137313990-student-training-in-engineering-program-step-2022/?degree=BACHELORS&distance=50&employment_type=INTERN&q=STEP), [Munich](https://careers.google.com/jobs/results/84791753938215622-student-training-in-engineering-program-step-2022/?degree=BACHELORS&distance=50&employment_type=INTERN&q=STEP), [Tel Aviv-Yafo](https://careers.google.com/jobs/results/142925493499241158-student-training-in-engineering-program-step-2022/?degree=BACHELORS&distance=50&employment_type=INTERN&q=STEP),  APAC - [Taipei](https://careers.google.com/jobs/results/80443299602539206-student-training-in-engineering-program-step-intern-2022/?degree=BACHELORS&distance=50&employment_type=INTERN&q=STEP), [Beijing](https://careers.google.com/jobs/results/79899759344526022-student-training-in-engineering-program-step-intern-2022/?degree=BACHELORS&distance=50&employment_type=INTERN&q=STEP)| STEP Internship for first and second year students. |
```

Comment:
Hello! Seems good, but could you please add it to the end of the list and amend this commit? @aedobrynin

Reply:
> Hello! Seems good, but could you please add both Google and Datadog it to the end of the list and amend this commit? @aedobrynin

Fixed